src/Sarif/Schemata/Sarif-1.0.0.schema.json

```diff
@@ -0,0 +1,1250 @@
+{
```

Comment:
This file is copied without change from Sarif.schema.json in the master branch to Sarif-1.0.0.schema.json in the sarif-v2 branch.